components/user-store/org.wso2.carbon.identity.user.store.count/src/main/java/org/wso2/carbon/identity/user/store/count/jdbc/JDBCUserStoreCountRetriever.java

```diff
@@ -36,8 +36,12 @@
 import java.sql.PreparedStatement;
 import java.sql.ResultSet;
 import java.sql.SQLException;
+
 import javax.sql.DataSource;
 
+/**
+ * Use for retrieve users and roles count from JDBC user store.
```

Comment:
This should be a valid sentence.
```suggestion
 * Use to retrieve users and roles count from the JDBC user store.
```